docker/neumann/Dockerfile

```diff
@@ -33,7 +37,11 @@ USER $USERNAME
 
 # Copy files
 COPY --chown=$USERNAME --from=builder /build/target/release/$BINARY  /$USERNAME/$BINARY
-COPY --chown=$USERNAME ./resources /$USERNAME/resources
```

Comment:
What was in `resources` before? I don't remember this line or if these were important files.

Reply:
it was part of the 2 step process to get the chain spec into the container.